set-web/src/main/java/eu/europeana/set/web/service/impl/UserSetServiceImpl.java

```diff
@@ -613,7 +614,7 @@ public CollectionPage buildCollectionPage(UserSet userSet, LdProfiles profile, i
 	String collectionUrl = buildCollectionUrl(null, request.getRequestURL().toString(), request.getQueryString());
 
 	CollectionOverview partOf = buildCollectionOverview(collectionUrl, pageSize, totalInCollection, lastPage,
-		CommonLdConstants.COLLECTION);
+		CommonLdConstants.COLLECTION, profile);
```

Comment:
after discussion with Hugo, the profile must not be added to partOf URLs

Reply:
please remove the profile parameter from this method

Reply:
done